### PR TITLE
feat: add course details page

### PR DESCRIPTION
**Pull request by pankaj-prog**
- feat: add course details page
- style: buttons and link text
- fix: netlify page not found issue on refresh

### CODE REVIEW

**Comment by pankaj-prog** [comment]
Preview link for course details page: https://deploy-preview-5--skillup-store.netlify.app/course

File: src/pages/CourseDetails/CourseDetails.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+const course = {
+  _id: uuid(),
+  title: "How to Raise Over $10M In Crowdfunding",
+  subTitle:
+    "Startup Survivor & Drone Pioneer, Thomas Agaraté Will Teach You the 'Copy-and-Paste' for Crowdfunding Success a $1 Million Dollar Success",
+  description:
+    "Customers are buying online more than ever and crowdfunding is at its peak. Don't wait! This detailed course explains startup survivor and drone pioneer, Thomas Agaraté's winning method from A to Z. It actually took him 5 years to learn and discover all the skills shown in this course, so whatever your marketing skills, you’ll get tons of value. You'll learn a tested and proven formula. Thomas' method has been used over and over again, with consistent awesome results. Campaigners who used this method raised 5 to 20x more. So if you wanna be one of them, this is the right place for you. Get simple tools and processes you can follow and execute like a professional.",
+  img: "https://github.com/pankaj-prog/Ecom-images/blob/main/Business/crowdfunding.jpg?raw=true",
+  originalPrice: "4999",
+  discountedPrice: "1899",
```
Comment (manojsarna) [inline src/pages/CourseDetails/CourseDetails.jsx:16]: Hosting images on github , thats unique 👍😉 , is it working great for you?

File: src/pages/CourseDetails/CourseDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          <div className="course-content-details">
+            <div className="bubble">
+              <span>Content</span>
+              <span className="h5">{course.duration}</span>
+            </div>
+            <div className="bubble">
+              <span>Lectures</span>
+              <span className="h5">{course.noOfLectures}</span>
+            </div>
+          </div>
```
Comment (manojsarna) [inline src/pages/CourseDetails/CourseDetails.jsx:50]: Rest of the code looks good!!

**Review by manojsarna — APPROVED** [review]
Overall the Ui looks pretty awesome

Github image hosting , Great Move 😉

File: src/pages/CourseDetails/CourseDetails.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+          <h4 className="course-title gutter-bottom-8">{course.title}</h4>
+          <p className="course-subtitle gutter-bottom-8">{course.subTitle}</p>
+          <p className="">
+            By <span className="text-link">{course.createdBy}</span> | in{" "}
+            <span className="text-link">{course.categoryName}</span>
+          </p>
+          <p className="course-rating gutter-bottom-16">
+            Average Rating: <span> {course.rating}/5 </span>{" "}
+          </p>
+          <h5 className="">Description</h5>
```
Comment (aiman-ansari) [inline src/pages/CourseDetails/CourseDetails.jsx]: As you are not giving any styling to description you can remove className .

File: src/pages/CourseDetails/CourseDetails.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+            <div className="bubble">
+              <span>Content</span>
+              <span className="h5">{course.duration}</span>
+            </div>
+            <div className="bubble">
+              <span>Lectures</span>
+              <span className="h5">{course.noOfLectures}</span>
+            </div>
+          </div>
+        </section>
```
Comment (aiman-ansari) [inline src/pages/CourseDetails/CourseDetails.jsx:51]: clean code.

**Review by aiman-ansari — APPROVED** [review]
Everything looks good.

**Review by saurabhchirde — APPROVED** [review]
As already mentioned by Manoj, liked the way you hosted images on GitHub. I will also think about it.
The rest code looks good to merge.

**Review by m26unkwn — APPROVED** [review]
Individual course section is good.
using Github for your image is also good.

File: src/pages/CourseDetails/CourseDetails.jsx (hunk 1 above)
Comment (pankaj-prog) [reply src/pages/CourseDetails/CourseDetails.jsx:16]: Yess

File: src/pages/CourseDetails/CourseDetails.jsx (hunk 2 above)
Comment (pankaj-prog) [reply src/pages/CourseDetails/CourseDetails.jsx:50]: Thanks

File: src/pages/CourseDetails/CourseDetails.jsx (hunk 3 above)
Comment (pankaj-prog) [reply src/pages/CourseDetails/CourseDetails.jsx]: sure

File: src/pages/CourseDetails/CourseDetails.jsx (hunk 4 above)
Comment (pankaj-prog) [reply src/pages/CourseDetails/CourseDetails.jsx:51]: Thanks

**Pull request merged**